### DISCUSSION TITLE
如何评价技术是否优雅

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#4] [General]
# 如何评价技术是否优雅

技术优雅的评价标准就是看该技术或者方案设计是否具备通用性，是否符合大众认同的标准，是否完备自洽、简单易用、良好可扩展、具备美感。技术的优雅自底向上分为下面几个层次：1. 代码优雅；2. 架构优雅；3. 设计优雅；

https://www.edony.ink/what-is-elegant-technology/

**Comment by edonyzpc** [comment]
平时的技术方案是不是经常听到一种评论“不是很优雅”，对于这个问题我最大的问题是：我应该怎么做才可以优雅？找到方案和技术的原则之后，就知道如何评价，再加上多多阅读优秀的项目，审美自然而成，这样我们距离优雅的技术就不远了
